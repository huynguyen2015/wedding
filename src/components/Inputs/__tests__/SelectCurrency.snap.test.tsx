import SelectCurrency from '../SelectCurrency/index'
import renderer from 'react-test-renderer'

describe('SelectCurrency', function () {
  let dataTable
  beforeEach(function () {})

  afterEach(function () {
    dataTable = null
  })

  it('To match snapshot select currency', async () => {
    dataTable = renderer
      .create(
        <SelectCurrency
          value={1}
          fieldKey={'currency'}
          onChange={() => {}}
          placeholder="Select currency"
          currencies={[]}></SelectCurrency>
      )
      .toJSON()

    expect(dataTable).toMatchSnapshot()
  })
})
