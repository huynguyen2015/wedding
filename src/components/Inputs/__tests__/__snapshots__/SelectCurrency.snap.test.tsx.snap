// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectCurrency To match snapshot select currency 1`] = `
<div
  className="ant-select ant-select-lg ant-select-single ant-select-allow-clear ant-select-show-arrow ant-select-show-search"
  onBlur={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  style={
    Object {
      "width": "100%",
    }
  }
>
  <div
    className="ant-select-selector"
    onClick={[Function]}
    onMouseDown={[Function]}
  >
    <span
      className="ant-select-selection-search"
    >
      <input
        aria-activedescendant="undefined_list_0"
        aria-autocomplete="list"
        aria-controls="undefined_list"
        aria-haspopup="listbox"
        aria-owns="undefined_list"
        autoComplete="off"
        className="ant-select-selection-search-input"
        onChange={[Function]}
        onCompositionEnd={[Function]}
        onCompositionStart={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onPaste={[Function]}
        readOnly={false}
        role="combobox"
        style={
          Object {
            "opacity": null,
          }
        }
        type="search"
        unselectable={null}
        value=""
      />
    </span>
    <span
      className="ant-select-selection-item"
      title="1"
    >
      1
    </span>
  </div>
  <span
    aria-hidden={true}
    className="ant-select-arrow"
    onMouseDown={[Function]}
    style={
      Object {
        "WebkitUserSelect": "none",
        "userSelect": "none",
      }
    }
    unselectable="on"
  >
    <span
      aria-label="down"
      className="anticon anticon-down ant-select-suffix"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="down"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
        />
      </svg>
    </span>
  </span>
  <span
    aria-hidden={true}
    className="ant-select-clear"
    onMouseDown={[Function]}
    style={
      Object {
        "WebkitUserSelect": "none",
        "userSelect": "none",
      }
    }
    unselectable="on"
  >
    <span
      aria-label="close-circle"
      className="anticon anticon-close-circle"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="close-circle"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
        />
      </svg>
    </span>
  </span>
</div>
`;
